### ISSUE TITLE
dll on windows

### ISSUE THREAD

**Issue by jwijffels** [mlverse/torch#58]
FYI
current version on windows fails to load appropriately the dll
It says it can not find lantern_TensorIndex_new which is n lantern.h

```
R version 4.0.1 (2020-06-06) -- "See Things Now"
Copyright (C) 2020 The R Foundation for Statistical Computing
Platform: x86_64-w64-mingw32/x64 (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type 'license()' or 'licence()' for distribution details.

R is a collaborative project with many contributors.
Type 'contributors()' for more information and
'citation()' on how to cite R or R packages in publications.

Type 'demo()' for some demos, 'help()' for on-line help, or
'help.start()' for an HTML browser interface to help.
Type 'q()' to quit R.

> library(torch)
Error: package or namespace load failed for ‘torch’:
 .onLoad failed in loadNamespace() for 'torch', details:
  call: cpp_lantern_init(lantern_install_path())
  error: lantern_TensorIndex_new - Kan opgegeven procedure niet vinden.
```

**Comment by dfalbel** [comment]
This will probably be fixed with #55

**Comment by jwijffels** [comment]
thanks. looks good!
